confluence/confluence-reports.yaml

```diff
@@ -115,9 +115,23 @@ wiki_page:
           report-description: "All graphs in Dashboard"
           attachment-filename: "dashboard-test-automation-coverage-fenix.png"
           looker-graph-url: "https://mozilla.cloud.looker.com/dashboards/562"
-          
+
         - report-title: "iOS Test Automation Coverage"
           report-description: "TBD"
           attachment-filename: "404.png"
           looker-graph-url: "https://mozilla.cloud.looker.com/dashboards/601"
 
+    - name: "Android UI Test Insights Dashboard"
+      reports:
+        - report-title: "Android UI Test Insights"
+          report-description: "All graphs in Dashboard"
+          attachment-filename: "404.png"
```

Comment:
For dashboards, we don't create a real time thumbnail, just a screenshot of the dashboard, so you can create one and add it to the images folder similar to `dashboard-test-automation-coverage-fenix.png`